calcom/cal.com
# fix: styles for the schedule component, make it mobile-friendly

## What does this PR do?



- Fixes CAL-3797, Make the Schedule component mobile-friendly.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



Visually check the Schedule component view on the /availability/ page.

before (wide screen)
<img width="887" alt="image" src="https://github.com/calcom/cal.com/assets/3423341/b0aa93b1-3938-4022-980f-91c3226e981c">

before (mobile view)
<img width="462" alt="image" src="https://github.com/calcom/cal.com/assets/3423341/a52b88f8-e75d-4ea0-873d-d3ff701c627e">

after  (wide screen)
<img width="962" alt="image" src="https://github.com/calcom/cal.com/assets/3423341/a8018a9b-7381-4cc8-b5e4-b2629d3fd996">

after (mobile view)
<img width="535" alt="image" src="https://github.com/calcom/cal.com/assets/3423341/986ef08b-603f-4aa6-b610-8d494469ec20">


design check (compare with Figma)
<img width="693" alt="image" src="https://github.com/calcom/cal.com/assets/3423341/9530b364-b77e-4fc4-8a71-d13d6a884ab2">
